nominees/ekshop.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "ekShop",
+  "aliases": [
+    ""
+  ],
+  "description": "ekShop is a fully open-source eCommerce platform that is customizable and configurable to your needs. It is completely free, adaptable and open to be supported by a worldwide community of volunteers and contributors. It’s free and open source nature allows users to maintain complete control of the data content and modify it as they wish and according to their needs. Our intended goal is to allow any potential entrepreneur to quickly get up and running with an online platform to sell goods online.",
+  "website": "https://ekshop.gov.bd/",
+  "license": [
+    {
+      "spdx": "Apache-2.0",
+      "licenseURL": "https://github.com/a2i-dpg/ekShop/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 1,
+      "evidenceText": "ekShop is a scalable open-source eCommerce solution that provides the power of harnessing online marketplace and businesses into the hands of everyday folks. ekShop allows people to bring forth sustainable livelihoods by engaging entrepreneurs, merchants, and employees that help the business operate. As a result, ekShop, when implemented correctly, can uplift communities and populations out of poverty."
+    },
+    {
+      "SDGNumber": 5,
+      "evidenceText": "ekShop has strict policies for preventing all forms of gender discrimination by offering equal opportunity to every gender, race, and ethnicity to participate in its platform - both in the capacity of buyer and seller. Women, in particular, can find the means of selling their creations and handicrafts through the platform and gain reach, unlike other means before. "
+    },
+    {
+      "SDGNumber": 9,
+      "evidenceText": "ekShop provides a new infrastructure hosted in the digital space and allows for developing sustainable business models in its platform. This fosters economic development for communities and populations as they engage in this new platform to conduct business and transact with finances.  As a result of this platform and the users it attracts, financial institutions are also brought forth to be made accessible to those who are participants of the platform, effectively increasing the scope of exposure for Financial Institutions to those who are needy and poor. ekShop's infrastructure depends on accessibility by the internet, and its creation and continued survival will ensure the development of internet connectivity in underdeveloped regions to expand its reach. "
+    },
+    {
+      "SDGNumber": 10,
+      "evidenceText": "ekShop allows participants to use the platform without requiring any major upfront fees or expenses, unlike a traditional brick-and-mortar shop.  As a result, ekShop ensures that the lower class of economically successful population can easily take a role in this platform and significantly improve their chances of increasing their income growth potential."
```

Comment:
I could possibly see their case here, but I'd like to push back on this SDG too and ask them to contextualize their claim against [SDG 10 targets and indicators](https://sdgs.un.org/goals/goal10).

Reply:
Hi @jwflory, @conradsp and @nathanbaleeta. Justin and Steve, I'm Ben, I am with the UNDPs DPG team and recently assigned to support our most promising digital products replicate and scale as DPGs. Glad to be connected!

Thanks for the good comments and questions. I've reached out to the team for them to add more information as to the impact of the ekShop platform, their impact in bringing rural women into the economy and value they bring to micro merchants in several LMICs. See attached. The team behind it has been on this for multiple years and gotten recognition both in Bangladesh, won UNs Innovation Facility Fund in 2018 and mentioned as a pioneer in digital inclusivity by UNDPs General Secretary.

As to the SDGs, the team is requesting to keep SDG 5 and change from SDG 10 to 9 'Industry, Innovation and Infrastructure'. It would be hugely helpful if you could update, and let me know if more is needed from my or the teams side? 

[MAJOR EKSHOP IMPACTS (2).pdf](https://github.com/DPGAlliance/publicgoods-candidates/files/8659306/MAJOR.EKSHOP.IMPACTS.2.pdf)

Reply:
@benbertelsen Hi, thanks for adding to the review here. Since the SDG section in the report is short, here is the excerpt for other reviewers:

> Goal 5 – Gender Equality
> ➔ Using women for end-mile logistics connection through the micro-merchants’ initiative.
> ➔ Progressive distribution of entrepreneurs of male to female (50:50).
> ➔ 77% of the rural products marketed through ekShop are produced by women
> ➔ 2000 micro-merchants (piloting in 4 districts) currently working with ekShop platforms are all women.
>
> Goal 9 - Industry, Innovation, and Infrastructure
> ➔ Innovative technology integrated by ekShop includes complete AI-based BI tools.
> ➔ Introduced country’s first ESCROW payment security.
> ➔ Countrywide geo-tag based logistic mapping to any address, be it without a postcode, is automatically mapped. The system will tell how long a parcel will take to reach there.
> ➔ UDC is often used as storage warehouses for products that are frequently ordered by locals.

To move this review forward and unblock the nomination, here is what we need from you and the ekShop team:

1. A paragraph summary of the relevance to [SDG 5](https://sdgs.un.org/goals/goal5) and [SDG 9](https://sdgs.un.org/goals/goal9). This is attached to the submission in the Registry if I recall correctly. If you can connect the relevance to specific Targets & Indicators of each SDG, this makes the review easier.
2. If you have a URL to a blog post, relevant research, or other useful context to your summary, please provide this too.

Once that is in place, @nathanbaleeta can update the submission on your behalf and we can clear this blocker from the review.

Reply:
Thank you @benbertelsen & @nathanbaleeta, this looks good to me!